include/RAJA/policy/hip/params/kernel_name.hpp

```diff
@@ -13,18 +13,19 @@
 
 namespace RAJA
 {
-namespace expt
-{
```

Comment:
Do the other functions in here like `param_init`, `param_combine`, and `param_resolve` still need to remain in the `expt` namespace?

Reply:
I think `expt` is meant for more outward facing RAJA calls, I don't think keeping them in expt adds to anything.

Reply:
@artv3 I made changes to these `kernel_name.hpp`s which leave the `expt` namespace in there, and scope `Name` more precisely. Please take a look when you can.